bookkeeper-server/src/main/java/org/apache/bookkeeper/server/http/BKHttpServiceProvider.java

```diff
@@ -79,6 +83,13 @@ private BKHttpServiceProvider(BookieServer bookieServer,
         this.bookieServer = bookieServer;
         this.autoRecovery = autoRecovery;
         this.serverConf = serverConf;
+        this.layoutManager = new ZkLayoutManager(
```

Comment:
Can you get the layout manager from bookieServer?

Reply:
Thanks, will try it